guides/docker.md

```diff
@@ -0,0 +1,52 @@
+# Docker
+
+## Running from DockerHub
+
+We provide prebuilt MongoosePush images. Configuration requires either an FCM token, APNS certificates or an APNS token. Depending on your usecase, you can have some or all of them in a standalone MongoosePush instance or using a docker container.
+For the full configuration you need to set up the following directory structure:
```

Comment:
I knew I've seen that before:
https://github.com/esl/MongoosePush/pull/103#discussion_r322683529
```suggestion
For the full configuration you need to set the following directory structure up:
```

Reply:
And I saw it in this PR already [HERE](https://github.com/esl/MongoosePush/pull/183#discussion_r448946353)!

Reply:
"set up" is correct here. @kmakiela, you originally had "setup", which is a noun, not a verb ("set up" vs "setup"), in the position of a verb. @goddammit corrected that to a "set something up" construction, which is correct, but in this particular example I find that inner "something" too long and pushes the two particles of the verb too far away from each other, so I thought of "set up something" instead.